README.md

```diff
@@ -237,6 +237,19 @@ data already loaded.
   rails server
   ```
 
+### API
```

Comment:
maybe call this title `API Documentation`?

Reply:
@kennyadsl done.